src/app/components/code-block/index.html

```diff
@@ -0,0 +1,56 @@
+    </p>
+    <p>
+      <sky-alert alertType="info">
+        To use the code block component, you must set it up as a dependency in your SKY UX SPA. First,install the package as part of your SPA dependencies: <stache-code>npm install @blackbaud/skyux-lib-code-block --save</stache-code>. Then install its plugin: <stache-code>npm install @blackbaud/skyux-builder-plugin-code-block --save</stache-code>. Next, add <stache-code>SkyCodeBlockModule</stache-code> as both an import and export in your <stache-code>app-extras.module.ts</stache-code> file. Finally, add the code block plugin to your <stache-code>skyuxconfig.json</stache-code> file <stache-code>"plugins": ["@blackbaud/skyux-builder-plugin-code-block"]</stache-code>.
+      </sky-alert>
+    </p>
+  </sky-demo-page-summary>
+
+  <sky-demo-page-properties>
+    <sky-demo-page-property
+      propertyName="code"
+      isOptional="true"
+    >
+      Passes a code sample into the component. This property binds data, so you can use brackets to evaluate the template expression: <stache-code>[code]</stache-code>. Without the brackets, Angular treats the value as a string and initializes the target property with the string value instead of evaluating the data value. This property accepts <stache-code>string</stache-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="hideCopyToClipboard"
+      isOptional="true"
+      defaultValue="false"
+    >
+      Indicates whether to hide the copy-to-clipboard button in the code-block header. This property accepts <stache-code>boolean</stache-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="languageType"
+      isOptional="true"
+      defaultValue="markup"
+    >
+      Indicates the language of the code sample. The language type highlights the syntax in your code sample and appears in the header. This property accepts <stache-code>string</stache-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="hideHeader"
+      isOptional="true"
+      defaultValue="false"
+    >
+      Indicates whether to hide the header portion of the code block. If the <stache-code>languageType</stache-code> is not declared and <stache-code>hideCopyToClipboard</stache-code> is set to true, the header is hidden. This property accepts <stache-code>boolean</stache-code> values.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-example>
+    <sky-code-block-demo></sky-code-block-demo>
+    <sky-demo-page-code demoName="Code block"></sky-demo-page-code>
```

Comment:
I agree with Trevor's thought that we should display something other than the code block element in the code block demo. That becomes kinda circular and doesn't really illustrate what this is for. ... I'm not sure if Trevor ever got back to you with suggestions for what to include in the code block, but if not, we could use something from the site. For example, the TypeScript in the `app-extras.module.ts` sample at https://developer.blackbaud.com/skyux/learn/get-started/advanced/localization#provide-a-custom-locale-provider. But if we do that, I'd add a comment to the top of the code block to explain the sample. For example, if you use the code block I point to, we could add something like this to the start:
```         
// This code block illustrates a custom implementation of SkyAppLocaleProvider in the app-extras.module.ts file.
// To see the code block in place within the SKY UX documentation website, go to:
// https://developer.blackbaud.com/skyux/learn/get-started/advanced/localization#provide-a-custom-locale-provider
```
Or something like that.